src/namicorn.test.ts

```diff
@@ -9,192 +9,229 @@ const MAINNET_URL = 'https://mainnet.infura.io';
 const ZILLIGA_URL = 'https://api.zilliqa.com';
 
 const mockAPICalls = (topLevel, testName, url = MAINNET_URL) => {
-	const mockCall = mockData[topLevel][testName];
-
-	mockCall.forEach(({ METHOD, REQUEST, RESPONSE }) => {
-		switch (METHOD) {
-			case 'POST': {
-				nock(url)
-					// .log(console.log)
-					.post('/', JSON.stringify(REQUEST))
-					.reply(200, JSON.stringify(RESPONSE));
-			}
-			default: {
-				nock(url)
-					// .log(console.log)
-					.get(REQUEST)
-					.reply(200, RESPONSE);
-			}
-		}
-	});
+  const mockCall = mockData[topLevel][testName];
+
+  mockCall.forEach(({ METHOD, REQUEST, RESPONSE }) => {
+    switch (METHOD) {
+      case 'POST': {
+        nock(url)
+          // .log(console.log)
+          .post('/', JSON.stringify(REQUEST))
+          .reply(200, JSON.stringify(RESPONSE));
+      }
+      default: {
+        nock(url)
+          // .log(console.log)
+          .get(REQUEST)
+          .reply(200, RESPONSE);
+      }
+    }
+  });
 };
 
 beforeEach(() => {
-	nock.cleanAll();
+  nock.cleanAll();
 });
 
 afterEach(() => {
-	if (!nock.isDone()) {
-		nock.cleanAll();
-	}
+  if (!nock.isDone()) {
+    nock.cleanAll();
+  }
 });
 
 it('should work', async () => {
-	nock.cleanAll();
-	const testName = 'should work';
-	mockAPICalls('UD_API', testName, DEFAULT_URL);
-	const namicorn = new Namicorn();
-	const result = await namicorn.address('cofounding.zil', 'eth');
-	expect(result).toEqual('0xaa91734f90795e80751c96e682a321bb3c1a4186');
+  nock.cleanAll();
+  const testName = 'should work';
+  mockAPICalls('UD_API', testName, DEFAULT_URL);
+  const namicorn = new Namicorn();
+  const result = await namicorn.address('cofounding.zil', 'eth');
+  expect(result).toEqual('0xaa91734f90795e80751c96e682a321bb3c1a4186');
 });
 
 it('resolves .eth name using blockchain', async () => {
-	const testName = 'resolves .eth name using blockchain';
-	mockAPICalls('ENS', testName, MAINNET_URL);
-
-	const namicorn = new Namicorn({
-		blockchain: { ens: MAINNET_URL },
-	});
-	var result = await namicorn.address('matthewgould.eth', 'ETH');
-	expect(result).toEqual('0x714ef33943d925731FBB89C99aF5780D888bD106');
+  const testName = 'resolves .eth name using blockchain';
+  mockAPICalls('ENS', testName, MAINNET_URL);
```

Comment:
You didn't convert this test to mock methods instead of API calls. Does it mock correctly?